### ISSUE TITLE
Feature request: Locking movement along axes

### ISSUE THREAD

**Issue by ColinGilbert** [DanielChappuis/reactphysics3d#21]
Hi,

I am working on a dynamic character controller, and it would be great to be able to lock linear/rotational movement along certain axes for a given object. Something along the line of 

```
body->setAngularFactor(rp3d::Vector3(0.0, 0.0, 1.0));
body->setLinearFactor(rp3d::Vector3(1.0, 0.0, 1.0))
```

Would be really, really handy.

Thanks!

**Comment by DanielChappuis** [comment]
Good idea ! I will try to add this in the next release.

**Comment by mokalux** [comment]
I am trying to set a body to a fixed rotation as well, something like body:setFixedRotation(bool) in box2d.
Is it possible to do something similar in current reactphysics3d?

Thank you.

**Comment by codenamecpp** [comment]
I hope this feature will be implemented one day :)

**Comment by DanielChappuis** [comment]
It's already implemented in the 'develop' branch. I will release the next version of the library in the next weeks with this feature.

**Comment by DanielChappuis** [comment]
I glad to announce that this feature is now implemented in version [v0.9.0](https://github.com/DanielChappuis/reactphysics3d/releases/tag/v0.9.0) that has just been released.

There is now a `RigidBody::setLinearLockAxisFactor()` method to lock the translational movement of a body along the world-space x, y and z axes and a `RigidBody::setAngularLockAxisFactor()` method to lock the rotational movement of a body around the world-space x, y and z axes.

Do not hesitate to take a look at the [documentation](https://reactphysics3d.com/documentation.html) for more information.
